config.proto

```diff
@@ -127,3 +130,17 @@ message JavaAgent {
     // filter_jar_paths is the list of path to filter jars, separated by `,`
     repeated google.protobuf.StringValue filter_jar_paths = 1;
 }
+
+// CustomDataCaptureEndpoint represents a custom data capture rule for an endpoint that matches
+// the given url pattern and host header
+message CustomDataCaptureEndpoint {
+
+    // data_capture describes the data being captured by instrumentation for this endpoint
+    DataCapture data_capture = 1;
+
+    // url_pattern describes the url pattern to match for this endpoint
+    google.protobuf.StringValue url_pattern = 2;
```

Comment:
we should rename it to `path_regex` (used by traceable agent) or `path_pattern`.

Reply:
How will this work for RPC?